certified-connectors/OzyApprovals/apiDefinition.swagger.json

```diff
@@ -1 +1,1285 @@
+                    "value-collection": "msg",
+                    "value-title": "name"
+                  }
+                },
+                "stage_id": {
+                  "type": "integer",
+                  "description": "",
+                  "title": "Approval Stage",
+                  "format": "string",
+                  "x-ms-dynamic-list": {
+                    "operationId": "GetProcessApprovalStages",
+                    "itemValuePath": "id",
+                    "itemTitlePath": "name",
+                    "parameters": {
+                      "process_id": {
+                        "parameterReference": "body/process_id"
+                      }
+                    }
+                  },
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetProcessApprovalStages",
+                    "value-path": "id",
+                    "value-collection": "msg",
+                    "value-title": "name",
+                    "parameters": {
+                      "process_id": {
+                        "parameter": "process_id"
+                      }
+                    }
+                  }
+                }
+              },
+              "required": [
+                "target_url",
+                "operation",
+                "process_id",
+                "stage_id"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/get_approval_processes": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Status"
+                },
+                "subject": {
+                  "type": "string",
+                  "description": "Subject"
+                },
+                "msg": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "integer",
+                        "description": "id",
+                        "title": "",
+                        "format": "string"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "Name"
+                      }
+                    }
+                  },
+                  "description": "msg"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Approval Processes",
+        "description": "Get Approval Processes",
+        "operationId": "GetApprovalProcesses",
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/get_approval_stages2/{process_id}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
Please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.